src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.AgentPool.json

```diff
@@ -162,6 +162,40 @@
         "AutomatedFix"
       ],
       "Enabled": true
+    },
+    {
+      "ControlID": "ADO_AgentPool_AuthZ_Enable_Branch_Control",
+      "Description": "Allow agent pools to be accessed only by select branches.",
+      "Id": "AgentPool210",
+      "ControlSeverity": "Medium",
+      "Automated": "Yes",
+      "MethodName": "CheckBranchControlForAgentPool",
+      "Rationale": "Once a agent pool is made accessible to a YAML pipeline, malicious users with 'create branch' permissions on the repository will be able to access the agent pool by queueing the pipeline from the branch they select even if they do not have access to the agent pool. To prevent this ensure that the agent pool can be accessed only from select branches (eg. 'main').",
```

Comment:
Once a agent pool --> Once an agent pool
(eg. 'main'). -- > (e.g., 'main').

Reply:
Did the changes